src/sql/parts/dashboard/common/actions.ts

```diff
@@ -134,20 +136,49 @@ export class PinUnpinTabAction extends Action {
 }
 
 export class AddFeatureTabAction extends Action {
-	private static readonly ID = 'addFeatureTab';
-	private static readonly LABEL = nls.localize('addFeatureTab', "Add a feature tab");
+	private static readonly ID = 'openInstalledFeatures';
+	private static readonly LABEL = nls.localize('openInstalledFeatures', "Open installed features");
 	private static readonly ICON = 'new';
 
+	private _disposables: IDisposable[] = [];
+	private _openedTabs: IDashboardTab[] = [];
+
 	constructor(
 		private _dashboardTabs: Array<IDashboardTab>,
 		private _uri: string,
-		@INewDashboardTabService private _newDashboardTabService: INewDashboardTabService
+		@INewDashboardTabService private _newDashboardTabService: INewDashboardTabService,
```

Comment:
personally I think `IDashboardTabService` may be a more general purpose interface name (where New tab is a possible function on the interface.